treeherder/etl/job_loader.py

```diff
@@ -131,6 +131,7 @@ def transform(self, pulse_job):
             platform_src = pulse_job[v] if v in pulse_job else default_platform
             x["job"][k] = self._get_platform(platform_src)
 
+        # XXX: We should handle this in a follow up
```

Comment:
Maybe this should be prefaced with ``TODO: `` and a bug number for the follow up?